packages/config/src/options.ts

```diff
@@ -198,6 +198,12 @@ const driverConfigOptions: Array<DriverConfigOption> = [
     defaultValue: 60000,
     validation: validate.isNumber,
     overrideLevel: 'any',
+  }, {
+    name: 'experimentalCspAllowList',
+    defaultValue: false,
+    validation: validate.validateAny(validate.isBoolean, validate.isSubsetOf('script-src-elem', 'script-src', 'default-src', 'sandbox', 'form-action', 'navigate-to')),
```

Comment:
Is `validateAny` doing the same thing here as `isOneOf`? We might not need a new function which would be nice 😄 .

```suggestion
    validation: validate.isOneOf(validate.isBoolean, validate.isSubsetOf('script-src-elem', 'script-src', 'default-src', 'sandbox', 'form-action', 'navigate-to')),
```

Reply:
@AtofStryker `isOneOf` is not the same as `validateAny`
`validateAny` takes additional validators as arguments, and runs the validators against the value, and if ANY of the validators pass, then the validation passes.

Reply:
ah that makes sense. Looking closer at it is looks like you cant really pass a validator in that has already been executed.

Reply:
That is correct. It accepts validator functions that are then run in sequence against the value provided to the configuration option. In this case `isSubsetOf` is a thunk that returns a validator.